.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--no-profile
```

Comment:
Better default so that spec results are visible without having to scroll up. Can be overridden by passing `--profile N` to rspec.